src/components/ButtonGroup/GroupButton/index.test.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+
+import GroupButton from '.';
+import { noop } from '../../../util';
+
+const Icon = () => <span>🌭</span>;
+const Wrapper = (props) => {
+  return <div {...props} data-test="snapshot-wrapper" />;
+};
```

Comment:
[should] component should be removed

Reply:
Whoops 😞 I did not see that this was being used and being passed to `GroupButton`. That's why I was suggesting it's removal. My fault, disregard.